entity-service-impl/src/main/java/org/hypertrace/entity/query/service/EntityQueryServiceImpl.java

```diff
@@ -86,26 +94,63 @@ public void execute(EntityQueryRequest request, StreamObserver<ResultSetChunk> r
     }
 
     //TODO: Optimize this later. For now converting to EDS Query and then again to DocStore Query.
-    Map<String, String> scopedAttrNameToEDSAttrMap = attrNameToEDSAttrMap.get(request.getEntityType());
-    Query query = EntityQueryConverter.convertToEDSQuery(request, scopedAttrNameToEDSAttrMap);
+    Query query = EntityQueryConverter
+        .convertToEDSQuery(request, attrNameToEDSAttrMap.get(request.getEntityType()));
     /**
      * {@link EntityQueryRequest} selections need to treated differently, since they don't transform
      * one to one to {@link org.hypertrace.entity.data.service.v1.EntityDataRequest} selections
      */
+    Map<String, String> scopedAttrNameToEDSAttrMap = attrNameToEDSAttrMap.get(request.getEntityType());
```

Comment:
nit: the map statement can be moved to top and can be re-used in the argument in above line

```
Map<String, String> scopedAttrNameToEDSAttrMap = attrNameToEDSAttrMap.get(request.getEntityType());
Query query = EntityQueryConverter.convertToEDSQuery(request, scopedAttrNameToEDSAttrMap);
```

Reply:
Merging this one, will address it in a subsequent pr